tests/nn/test_inference.py

```diff
@@ -1381,3 +1386,63 @@ def test_flow_tracker(centered_pair_predictions: Labels, tmpdir):
         for key in tracker.candidate_maker.shifted_instances.keys():
             assert lf.frame_idx - key[0] <= track_window  # Keys are pruned
             assert abs(key[0] - key[1]) <= track_window  # References within window
+
+
+def test_movenet_inference(movenet_video):
+    inference_layer = MoveNetInferenceLayer(model_name="lightning")
+    inference_model = MoveNetInferenceModel(inference_layer)
+
+    p = sleap.pipelines.Pipeline(
+        sleap.pipelines.VideoReader(video=movenet_video, example_indices=[0])
+    )
+    p += sleap.pipelines.SizeMatcher(
+        points_key=None,
+        max_image_width=inference_model.image_size,
+        max_image_height=inference_model.image_size,
+        center_pad=True,
+    )
+    p += sleap.pipelines.Batcher(batch_size=4)
+
+    ex = p.peek(1)
+    preds = inference_model.predict_on_batch(ex)
+    assert preds["instance_peaks"].shape == (1, 1, 17, 2)
+
+
+def test_movenet_predictor(min_dance_labels, movenet_video):
+    predictor = MoveNetPredictor.from_trained_models("thunder")
+    predictor.verbosity = "none"
+    assert predictor.is_grayscale == False
+    labels_pr = predictor.predict(min_dance_labels)
+
+    vr = sleap.pipelines.VideoReader(video=movenet_video, example_indices=[0, 1, 2])
+    labels_pr = predictor.predict(data=vr)
+
+    assert len(labels_pr) == 3
+    assert len(labels_pr[0].instances) == 1
+
+    points_gt = np.concatenate(
+        [min_dance_labels[0][0].numpy(), min_dance_labels[1][0].numpy()], axis=0
+    )
+    points_pr = np.concatenate(
+        [labels_pr[0][0].numpy(), labels_pr[1][0].numpy()], axis=0
+    )
+
+    max_diff = np.nanmax(np.abs(points_gt - points_pr))
+    assert max_diff < 0.1
+
+
+# TODO (Jiaying): add a test for test_load_model. Leaving this part out for now as
+# Predictor.from_trained_models returns None not only for MoveNet but also other models.
+def test_load_model(min_dance_labels, movenet_video, model_name="ligntning"):
+    predictor = Predictor.from_trained_models(
+        model_name,
```

Comment:
This should be called using a keyword, otherwise it is used as a positional argument and thinks you are supplying the first argument of `Predictor.from_trained_models` (which is `model_paths`)
```suggestion
        model_name=model_name,
```